lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -152,8 +152,27 @@ sub group_overview {
     return $self->reply->not_found unless $group;
 
     my $res = $self->_group_result($group, $limit_builds);
-    $self->stash('result', $res);
-    $self->stash('group',  $group);
+    my @comments = $group->comments->all;
+    for my $comment (@comments) {
+        my @tag   = $comment->tag;
+        my $build = $tag[0];
+        next unless $build;
+        $self->app->log->debug('Tag found on build ' . $tag[0] . ' of type ' . $tag[1]);
+        $self->app->log->debug('description: ' . $tag[2]) if $tag[2];
+        if ($tag[1] eq '-important') {
+            $self->app->log->debug('Deleting tag on build ' . $build);
+            undef $res->{$build}->{tag};
```

Comment:
It's not clear to me what you want here. Do you want `$res->{$build}->{tag}` to exists and be undef? If not, use `delete` instead.